src/summary/history/allergy-card-level-three.component.tsx

```diff
@@ -0,0 +1,225 @@
+import React from "react";
+import { match } from "react-router";
+import SummaryCard from "../cards/summary-card.component";
+import style from "./allergy-card-level-three.css";
+import {
+  getAllergyAllergenByConceptUuid,
+  getAllergyReaction
+} from "./allergy-intolerance.resource";
+import { createErrorHandler } from "@openmrs/esm-error-handling";
+
+const DRUGALLERGENCONCEPT: string = "162552AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const ENVIROMENTALALLEGENCONCEPT: string =
+  "162554AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+const FOODALLERGENCONCEPT: string = "162553AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA";
+
+export function AllergyCardLevelThreeAdd(props: AllergyCardLevelThreeAddProps) {
```

Comment:
Are we using this component for both adding and editing? Perhaps just call it AllergyForm?

Reply:
I am using it for Adding an Allergy, should I combine the functionality to one component?